docs_guidelines/how_to/add_sections.rst

```diff
@@ -0,0 +1,93 @@
+=============================
+Add sections to documentation
+=============================
+
+This guide shows how to separate your documents in parts. 
+
+Pre-requisites
+==============
+
+This guide assumes your Qiskit project already has a working Sphinx documentation project in a folder called ``docs``. If you don't have it, you can set it up with
+`sphinx-quickstart <https://www.sphinx-doc.org/en/master/man/sphinx-quickstart.html>`_.
+
+You can organize your documents into:
+
+* Sections.
+* Subsections.
+* Subsubsections.
+* Paragraphs.
+
+Create sections
+===============
+
+In order to create a new section you can write the name of the section covered below by equal signs ``=``. For example,
+you can include this header:
+
+.. code-block:: text
+
+    Section name
+    ============
+
+And it would look like this:
+
+Section name
+============
+
+Create subsections
+==================
+
+You can add a subsection to your section by writing the corresponding name covered below by a sign. This time it's the hyphen, ``-``. So you can include, for example, this subsection header:
+
+.. code-block:: text
+
+    Subsection name
+    ---------------
+
+A subsection needs to be written as part of a section. That means the subsection header can't be shown on its own.
+
+Section name
+============
+Subsection name
+---------------
+
+Create subsubsections
+=====================
+
+If you want to add a subsubsection to a section of your document, you can do so by writing its title covered below by carets, ``^``. For example, you can add this subsubsection header:
+
+.. code-block:: text
+
+    Subsubsection name
+    ^^^^^^^^^^^^^^^^^^
+
+A subsubsection needs to be part of a subsection, so the proper structure would be:
+
+Section name
+============
+Subsection name
+---------------
+Subsubsection name
+^^^^^^^^^^^^^^^^^^
```

Comment:
Not really. At first I only wanted to show how each type was rendered after showing the code. But then I stumbled upon the problem that a subsubsection had to be included as part of a subsection so I couldn't show it alone.

I also thought about showing the full structure as both code and rendered but believed it would make the guide look too bloated so I stuck to showing the code in one line and then the rendered version with the other (necessary) headers